src/api/status/public/js/serviceStatus.js

```diff
@@ -0,0 +1,65 @@
+const reportTbody = () => document.querySelector('#report');
+const requestMessageReport = () => document.querySelector('#request-message');
+
+// Create td for service name
+const createTdText = (content) => {
+  const td = document.createElement('td');
+  const div = document.createElement('div');
+  const h6 = document.createElement('h6');
+
+  h6.className = 'text-capitalize mb-0 text-sm';
+  h6.innerHTML = content;
+  div.className = 'text-start px-2 py-1';
+  div.appendChild(h6);
+  td.appendChild(div);
+
+  return td;
+};
+
+// Create td for icon
+const createTdIcon = (status) => {
+  const td = document.createElement('td');
+  const icon = document.createElement('i');
+
+  td.className = 'align-middle text-center';
+  icon.title = status;
+  icon.className =
+    status >= 200 && status <= 299
+      ? 'text-lg text-success fas fa-check-circle'
+      : 'text-lg text-danger fas fa-exclamation-circle';
+
+  td.appendChild(icon);
+
+  return td;
+};
+
+// Create row of the table
+const serviceRowFormat = (result) => {
+  const tr = document.createElement('tr');
+
+  tr.appendChild(createTdText(result.name));
+  tr.appendChild(createTdIcon(result.status.staging));
+  tr.appendChild(createTdIcon(result.status.production));
```

Comment:
```suggestion
  const {
    name,
    status: { staging, production },
  } = result;
```

Reply:
Fixed, I always forgot about destructuring assignment if I got the term right. Thanks, I will remind myself to do this in the future.